cpp/ql/test/library-tests/dataflow/DefaultTaintTracking/annotate_sinks_only/defaulttainttracking.cpp

```diff
@@ -19,7 +19,7 @@ int main() {
 
   char untainted_buf[100] = "";
   char buf[100] = "VAR = ";
-  sink(strcat(buf, getenv("VAR"))); // $ ast,ir
+  sink(strcat(buf, getenv("VAR"))); // $ ast MISSING: ir
```

Comment:
Is this result recoverable?

Reply:
I hope so! I haven't quite understood why we lose this result yet. But since we catch the flow to `buf` a few lines down I'm assuming that there's a simple fix for it (although I spend some time this afternoon looking for that fix, but didn't find it).

Reply:
Oh! Now I know why we lose this result. It's due to the extra sanitizers that were added in an early state of this PR: https://github.com/github/codeql/commit/fbe857d1fa8a3b577e043a74155b804214522373#diff-ab19ecc0bd6242557ac1c5c146893c5295950d847c2d3b216391712bbb5f76eeR224. I think we should accept this lost result for now and revisit it later.